src/inc/clrnt.h

```diff
@@ -948,7 +948,11 @@ typedef struct _DISPATCHER_CONTEXT {
                                                             
 // This function returns the length of a function using the new unwind info on arm.
 // Taken from minkernel\ntos\rtl\arm\ntrtlarm.h.
+#ifdef _MSC_VER_
```

Comment:
I added the `FORCEINLINE` definition to PAL, so it can always be defined as such.